storage/postgres/storage_test.go

```diff
@@ -11,35 +11,165 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
 package postgres
 
 import (
+	"bytes"
 	"context"
+	"crypto"
+	"crypto/sha256"
 	"database/sql"
 	"flag"
+	"fmt"
 	"os"
 	"testing"
-	"time"
 
 	"github.com/golang/glog"
+	"github.com/google/trillian"
+	tcrypto "github.com/google/trillian/crypto"
+	"github.com/google/trillian/storage"
 	"github.com/google/trillian/storage/postgres/testdb"
+	storageto "github.com/google/trillian/storage/testonly"
+	"github.com/google/trillian/testonly"
+	"github.com/google/trillian/types"
 )
 
-// db is shared throughout all postgres tests
-var db *sql.DB
```

Comment:
The flow of tests demanded this be here instead of there.  I'm still fresh to GO, so I don't have all the whys down, but, basically, it was being referenced before admin_storage_test.go was called.  At least, that's what seemed to be happening and moving it fixed the issue.